dashboard-app/src/hooks/useErrorsEffect.ts

```diff
@@ -0,0 +1,21 @@
+import { Dispatch, SetStateAction, useEffect } from 'react';
+import { AxiosError } from 'axios';
+import { AggregatedData } from '../dashboard/dataManipulation/logsToAggregatedData';
+import { Dictionary } from 'ramda';
+
+export const useErrorsEffect = (
+  error: AxiosError | undefined,
```

Comment:
`error?:` and `data?:` instead of `| undefined`

Reply:
`?` makes it an optional argument, which messes with arg order